### ISSUE TITLE
Update Offcanvas options after open

### ISSUE THREAD

**Issue by MatDepInfo** [ng-bootstrap/ng-bootstrap#4396]
Hello,
I don't achieve to modify panelClass of the offcanvas after I open it. Is there a way to do this ? If not, it will be great !

**Comment by maxokorokov** [comment]
Similar to #4281 for modal

**Comment by MatDepInfo** [comment]
Indeed, I achieve it with: ._panelCmptRef.instance.panelClass. But I have to declare _panelCmptRef as public in the vendor file...
